src/comp/shareButtonData/data.tsx

```diff
@@ -64,13 +59,28 @@ export const Url = ({ url, introduction, title }: UrlProps) => {
       definition: faGetPocket,
       url: `http://getpocket.com/edit?url=${url}&title=${title}`,
     },
-    Copy: {
-      name: "Copy",
-      definition: faLink,
-      url: url,
-    },
   };
-  return iconsUrl;
+  return iconsData;
+};
+export const ReturnShareSet = (props: UrlProps) => (needed: NeededIcon) => {
+  const returnSet: ShareButtonSet[] = [];
+  const iconsData = ShareSet(props);
+  if (needed === "main") {
+    mainIconName.map((iconName: IconName) =>
+      returnSet.push(iconsData[iconName])
+    );
```

Comment:
```suggestion
    const mainSet = mainIconName.map((name: IconName) => iconsData[name]);
    returnSet.push(...mainSet);
```
こっちの方が分かりやすい(個人的意見ではあるが)